### PR TITLE
[_]:(feature) New complete survey referral step

### PR DESCRIPTION
**Pull request by PixoDev**
New referral step, complete survey.

Now the app refresh the storage and referral links on app focus when it comes back from background to foreground

### CODE REVIEW

File: src/services/NetworkService/realtimeUpdates.ts [hunk 1]
```diff
@@ -1,33 +1,59 @@
...
-    const URL = `${constants.DRIVE_API_URL}`;
-    this.socket = io(URL, {
+  constructor(private sdk: SdkManager) {}
+
+  init() {
+    this.socket = io(constants.NOTIFICATIONS_URL, {
+      auth: {
+        token: this.sdk.getApiSecurity().photosToken,
+      },
       transports: ['websocket'],
```
Comment (sg-gs) [inline src/services/NetworkService/realtimeUpdates.ts:15]: remove this, as the we use the default transport

File: src/services/NetworkService/realtimeUpdates.ts [hunk 2]
```diff
@@ -1,33 +1,59 @@
...
 
-  constructor() {
-    const URL = `${constants.DRIVE_API_URL}`;
-    this.socket = io(URL, {
+  constructor(private sdk: SdkManager) {}
+
+  init() {
+    this.socket = io(constants.NOTIFICATIONS_URL, {
+      auth: {
+        token: this.sdk.getApiSecurity().photosToken,
```
Comment (sg-gs) [inline src/services/NetworkService/realtimeUpdates.ts:13]: the naming is a bit confusing, is not a `photosToken`, is the new auth token with the proper format.

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: src/services/NetworkService/realtimeUpdates.ts (hunk 2 above)
Comment (PixoDev) [reply src/services/NetworkService/realtimeUpdates.ts:13]: I know, but the name was already this when I started on mobile, is on the list of pending refactors, is maintained for compatibility and consistency, but will be definitely refactorized in the near future

File: src/services/NetworkService/realtimeUpdates.ts (hunk 1 above)
Comment (PixoDev) [reply src/services/NetworkService/realtimeUpdates.ts:15]: doesnt work well on mobile, seems like it defaults to xhr polling, and it throws connection errors

**Review by sg-gs — APPROVED** [review]

**Pull request merged**